### PR TITLE
Update color palette and button styles

### PR DESCRIPTION
**Pull request by orangejenny**
Based on @ohmegasquared 's work on the style guide.

FYI @dimagi/product  these changes are on staging if you'd like to poke around.

@dannyroberts / @biyeun 

mobile workers before
screen shot 2018-12-07 at 6 10 09 pm

mobile workers after
screen shot 2018-12-07 at 6 10 21 pm

organization structure before
screen shot 2018-12-07 at 6 11 07 pm

organization structure after
screen shot 2018-12-07 at 6 11 14 pm

app manager releases before
screen shot 2018-12-07 at 6 11 37 pm

app manager releases after
screen shot 2018-12-07 at 6 11 45 pm

### CODE REVIEW

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less [hunk 1]
```diff
@@ -23,17 +23,23 @@
 @cc-dark-warm-accent-mid: #ff8400;
 @cc-dark-warm-accent-low: #994f00;
 
-@cc-light-warm-accent-hi:   #f8ecbd;
-@cc-light-warm-accent-mid:  #f9c700;
-@cc-light-warm-accent-low:  #685300;
+@cc-light-warm-accent-extra-hi:   #fcf2cd;
```
Comment (biyeun) [inline corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less]: obv not actionable for this PR. but how is the feeling for the naming convention of `light-warm-accent` etc...not sure if this is clearer than color names?

**Comment by biyeun** [comment]
😍

**Review by biyeun — APPROVED** [review]

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less]: I think it makes sense for us to have a full palette of colors with designy names like `light-warm-accent` but also that devs writing application-level styles should usually be using more semantic names. So I've been thinking about leaving the `light-warm-accent`-type names but just adding more semantic synonyms to this file, like `@cc-link: @cc-brand-mid;` - what do you think?

**Comment by orangejenny** [comment]
Added a couple of commits tweaking individual pages and updating the login/registration/invitation flows:
screen shot 2018-12-11 at 9 34 16 pm

screen shot 2018-12-11 at 9 34 06 pm

screen shot 2018-12-11 at 9 33 51 pm

**Comment by biyeun** [comment]
non actionable, but with the switch, it feels like the text with the white background is more prominent than the text with the blue...

**Review by biyeun — APPROVED** [review]

**Pull request merged**